src/llmcompressor/modifiers/quantization/gptq/base.py

```diff
@@ -250,6 +258,11 @@ def apply_compression(
         logger.info(
             f"Running {class_name} calibration with " f"{len(dataloader)} samples..."
         )
+
+        # quantization scales and zp are already initialized but we do not
+        # want to calibrate wrt to these
+        self.model.apply(disable_quantization)
+
```

Comment:
One thought here: in the case where we have activation quantization and GPTQ on the weights, we will end up running calibration twice and I don't think we need to anymore. Since we aren't using the scale/zp during the Hessian calculation maybe we should try and set them in the same calibration as the hessian.

Not a requirement for now but just something to think about for the future.